src/FsAutoComplete/FsAutoComplete.Lsp.fs

```diff
@@ -1915,22 +1903,23 @@ type FSharpLspServer(backgroundServiceEnabled: bool, state: State, lspClient: FS
                   Command = ""
                   Arguments = None }
 
-              return { p with Command = Some cmd } |> success
+              return { p with Command = Some cmd } |> Some |> success
           else
-            let! res = commands.SymbolUseProject tyRes pos lineStr
+            let! res =
+              commands.SymbolUseProject tyRes pos lineStr
+              |> AsyncResult.ofCoreResponse
 
             let res =
               match res with
-              | CoreResponse.InfoRes msg
-              | CoreResponse.ErrorRes msg ->
+              | Core.Result.Error msg ->
                 logger.error (
                   Log.setMessage "CodeLensResolve - error getting symbol use for {file}"
                   >> Log.addContextDestructured "file" file
                   >> Log.addContextDestructured "error" msg
                 )
 
-                { p with Command = None } |> success
-              | CoreResponse.Res (LocationResponse.Use (sym, uses)) ->
+                success None
```

Comment:
this _seems_ to work and erase the codelens (which is better than the !!!MISSING COMMAND!!! text IMO)